lib/frameworks/cucumber.js

```diff
@@ -185,12 +196,17 @@ module.exports.run = function(config, specs, capabilities) {
         if (config.cucumberOpts.noSnippets) {
             execOptions.push('--no-snippets');
         }
+
+        if (config.cucumberOpts.ignoreUndefinedDefinitions) {
+            reporterOptions.ignoreUndefinedDefinitions =
+                config.cucumberOpts.ignoreUndefinedDefinitions;
+        }
     }
 
     global.cucumber = Cucumber.Cli(execOptions);
     cucumberConf = Cucumber.Cli.Configuration(execOptions);
     runtime = Cucumber.Runtime(cucumberConf);
-    reporter = new CucumberReporter(Cucumber.Listener(), capabilities);
+    reporter = new CucumberReporter(Cucumber.Listener(), capabilities, reporterOptions);
```

Comment:
let's simplify this and pass `config` instead of capabilities to the CucumberReporter. Then we can get rid of that if statement too.